vendor/engines/bulk_email/app/views/bulk_email/bulk_email/_export.html.haml

```diff
@@ -1,10 +1,12 @@
-= form_tag facility_bulk_email_path, :method => :get, :id => "bulk_email_export" do
+= form_tag facility_bulk_email_path, method: :get, id: "bulk_email_export" do
   = hidden_field_tag :format, :csv
-  = hidden_field_tag :search_type, @search_fields[:search_type]
+  - @search_fields[:bulk_email][:user_types].each do |user_type|
```

Comment:
Take a look at the HTML output on the Facility or Product edit forms. Most of those checkboxes are aligned properly.